tutorials/config.md

```diff
@@ -22,8 +22,11 @@ export const production = {
     };
   }
 };
+
 ```
 
+```general``` represent the name of your config.
```

Comment:
What does general mean here?

Reply:
It's the key word, or the variable that we try to configure I think? 

First time i read it, I didn't understand that it will be available on `config.general`

Reply:
only the configuration settings from the file `src/config/api.ts` are in the `config.general` namespace because that file exports into the `general`  namespace https://github.com/actionhero/actionhero/blob/master/src/config/api.ts#L4-L5